docs/source/asr/speaker_diarization/datasets.rst

```diff
@@ -1,72 +1,134 @@
 Datasets
 ========
-Check out page :doc:`Speech Classification Datasets <../speech_classification/datasets>` and :doc:`Speaker Recogniton Datasets <../speaker_recognition/datasets>` 
-for preparing datasets for training and validating VAD and speaker embedding models respectively.
 
-For Speaker Diarization inference, ``diarizer`` expects following arguments :
+This page is about preparing input dataset for diarization inference. To train or fine-tune speaker diarization system, the speaker embedding extractor should be trained or fine-tuned. Check out page :doc:`Speech Classification Datasets <../speech_classification/datasets>` and :doc:`Speaker Recogniton Datasets <../speaker_recognition/datasets>` 
+for preparing datasets for training and validating VAD and speaker embedding models respectively. 
 
-  - paths2audio_files
-  - oracle_num_speakers
-  - path2groundtruth_rttm_files [Optional]
-  - oracle_vad_manifest (if vad/rttm files are known)[Optional but suggested for good performace. This avoids FPR and TPR]
 
-paths2audio_files:
-  - a file containing absolute paths to audio files or list of paths to audio files. 
-  For generating a file that contains paths to audio files, you can simply use ``find`` bash command as shown below:
+Preparation of input data
+-------------------------
+
+Diarization inference is based on Hydra configurations which are fulfilled by .yaml files. See `NeMo Speaker Diarization Configuration Files <../configs>`_ for setting up the input configuration file for speaker diarization. Input data should be provided in line delimited JSON format as below:
```

Comment:
for setting up the input configuration file -> for setting up the input hydra configuration file. 

In order to remain consistent.

Reply:
Added "for setting up the input Hydra configuration file."